<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="3,9">
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>连接</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>连接到服务器</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ipt_address"/>
        </item>
        <item>
         <widget class="QPushButton" name="btn_connect">
          <property name="text">
           <string>连接</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>状态</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="3" column="1">
         <widget class="QSpinBox" name="status_pid_0">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QCheckBox" name="status_sensor3">
          <property name="text">
           <string>灰度4</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QCheckBox" name="status_sensor0">
          <property name="text">
           <string>灰度1</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="status_speed_1">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QCheckBox" name="status_sensor4">
          <property name="text">
           <string>灰度5</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QCheckBox" name="status_sensor2">
          <property name="text">
           <string>灰度3</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>速度左/速度右</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QCheckBox" name="status_sensor6">
          <property name="text">
           <string>人体2</string>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QSpinBox" name="status_speed_0">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QSpinBox" name="status_speed_2">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="status_sensor1">
          <property name="text">
           <string>灰度2</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="status_sensor5">
          <property name="text">
           <string>人体1</string>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>定时器</string>
          </property>
         </widget>
        </item>
        <item row="4" column="4">
         <widget class="QSpinBox" name="status_timer">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximum">
           <number>1000000</number>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QCheckBox" name="status_sensor7">
          <property name="text">
           <string>人体3</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>设置速度</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="pushButton_2">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>启动</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QSpinBox" name="status_pid_1">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QSpinBox" name="status_pid_2">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>PID</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="3,6">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>数据</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QListWidget" name="listWidget"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>图表</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QCustomPlot" name="plot" native="true"/>
        </item>
        <item row="0" column="1">
         <widget class="QScrollBar" name="verticalScrollBar">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QScrollBar" name="horizontalScrollBar">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>调试</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_4">
       <property name="text">
        <string>查看一小时数据表</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_5">
       <property name="text">
        <string>查看三日数据表</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>用户管理页面</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
